### PR TITLE
Update toast class names

### PR DESCRIPTION
**Pull request by strikerrocker**

### CODE REVIEW

File: mappings/net/minecraft/client/particle/CrackParticle.mapping [hunk 1]
```diff
@@ -10,7 +10,7 @@ CLASS dbs net/minecraft/client/particle/CrackParticle
 			ARG 2 x
 			ARG 3 y
 			ARG 5 velocityX
-	CLASS dbs$b
+	CLASS dbs$b FactorySlimeball
```
Comment (SquidDev) [inline mappings/net/minecraft/client/particle/CrackParticle.mapping]: I'd be inclined to call this `SlimeballFactory`, as it reads nicer (and same for the snowball below). Probably worth waiting to see what others think though,

Comment (asiekierka) [reply mappings/net/minecraft/client/particle/CrackParticle.mapping]: >FactorySlimeball

No.

SlimeballFactory etc

Comment (strikerrocker) [reply mappings/net/minecraft/client/particle/CrackParticle.mapping]: What abt FactoryItem?

**Comment by JamiesWhiteShirt** [comment]
I would suggest discarding the few particle changes you have, as I have a comprehensive mapping coming up very soon. We can discuss factory naming there.

**Comment by strikerrocker** [comment]
Looking for name suggestions for class_370

**Review by shadowfacts — APPROVED** [review]

**Pull request merged**